bench/vibrato-bench/src/main.rs

```diff
@@ -0,0 +1,40 @@
+use std::env;
+use std::fs::File;
+use std::io::{BufRead, BufReader};
+
+use vibrato::{Dictionary, Tokenizer};
+
+fn main() {
+    let args: Vec<String> = env::args().collect();
```

Comment:
Could you use clap instead of parsing arguments manually?

Reply:
https://github.com/legalforce-research/tokenizer-speed-bench/pull/2/commits/bc813a5df89be0697b84b5341d5b88597748442f